### PR TITLE
Addtime in sec

### PR DESCRIPTION
**Pull request by alex-Arc**

### CODE REVIEW

File: apps/server/src/controllers/integrationController.ts [hunk 1]
```diff
@@ -174,6 +174,19 @@ export function dispatchFromAdapter(
...
+    case 'addtime': {
+      const time = Number(payload);
+      if (isNaN(time)) {
+        throw new Error(`Time not recognised ${payload}`);
+      }
+      try {
+        PlaybackService.addTime(time);
+      } catch (error) {
+        throw new Error(`Could not add time: ${error}`);
+      }
```
Comment (cpvalente) [inline apps/server/src/controllers/integrationController.ts:186]: I see this is a pattern we use here, we should make a todo to improve this. In this case, re-throwing the error does not add any extra information or value

File: apps/server/src/controllers/integrationController.ts [hunk 2]
```diff
@@ -174,6 +174,19 @@ export function dispatchFromAdapter(
       PlaybackService.roll();
       break;
     }
+    case 'addtime': {
```
Comment (cpvalente) [inline apps/server/src/controllers/integrationController.ts:177]: I wonder if `addtime` is the best way to call it. 
I realise it matches the internal datapoint, but that doesnt have to be explicit to consumers.

However I dont have a better idea, what do you think?

File: apps/server/src/services/PlaybackService.ts [hunk 3]
```diff
@@ -269,17 +269,26 @@ export class PlaybackService {
     }
   }
 
+  /**
+   * Adds time to current event
+   * @param {number} time time to add in seconds
```
Comment (cpvalente) [inline apps/server/src/services/PlaybackService.ts:274]: you wrote time twice

File: apps/server/src/services/PlaybackService.ts [hunk 4]
```diff
@@ -269,17 +269,26 @@ export class PlaybackService {
...
+      eventTimer.addTime(timeInMs);
+      timeInMs > 0
+        ? logger.info(LogOrigin.Playback, `Added ${time} sec`)
+        : logger.info(LogOrigin.Playback, `Removed ${time} sec`);
+    }
+  }
+
   /**
    * Adds delay to current event
+   * @deprecated Use addTime
```
Comment (cpvalente) [inline apps/server/src/services/PlaybackService.ts:288]: great use of jsdocs, good stuff

**Review by cpvalente — COMMENTED** [review]
Code looks good, just some nitpicks and questions

File: apps/server/src/services/PlaybackService.ts (hunk 3 above)
Comment (alex-Arc) [reply apps/server/src/services/PlaybackService.ts:274]: ah, it formats like this for me... should I change it
image
image

Comment (cpvalente) [reply apps/server/src/services/PlaybackService.ts:274]: I usually add a - but didnt remember this was the reason 😅 
up to you, its all good

File: apps/server/src/controllers/integrationController.ts (hunk 2 above)
Comment (alex-Arc) [reply apps/server/src/controllers/integrationController.ts:178]: I am also a bit unsure
as I think this function is mostly for the speaker timer the convention from other apps like stagetimer.io seams to be add/remove time

**Comment by cpvalente** [comment]
Hi @alex-Arc , this looks good to me, lets line up the companion changes and we set this up for next release

**Pull request merged**